tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -773,7 +773,7 @@ Meta	;	cs	;	modern	;	Czech
 Meta	;	cy	;	modern	;	Welsh
 Meta	;	da	;	modern	;	Danish
 Meta	;	de	;	modern	;	German
-Meta	;	en	;	comprehensive	;	American English
+Meta	;	en	;	modern	;	American English
```

Comment:
When did we decide to move en to modern coverage?

Reply:
We decided some months ago. It doesn't affect vetting at all, because editing en is not permitted, and the charts are cleaner.